etc/config/c++.amazon.properties

```diff
@@ -318,6 +318,7 @@ group.armclang32.compilers=armv7-clang900:armv7-clang901:armv7-clang1000:armv7-c
 group.armclang32.isSemVer=true
 group.armclang32.compilerType=clang
 group.armclang32.supportsExecute=false
+group.armclang32.instructionSet=arm32
```

Comment:
Love it, thanks. I'm guessing there's some GCC equivalents to worry about too? If it's easy can you add them too.

Reply:
I went through the C/C++ configuration files and added the `arm32` `instructionSet` where it seemed obvious.  I'm not familiar with most of the others, and don't have an easy way to test the changes, so I'd rather not change any more.